wgengine/wgengine.go

```diff
@@ -89,10 +89,12 @@ type Engine interface {
 	// new Engine.
 	Close()
 
-	// Wait waits until the Engine's Close method is called or the
-	// engine aborts with an error. You don't have to call this.
+	// Wait returns a channel that is closed when the Engine's
+	// Close method is called, the engine aborts with an error,
+	// or it shuts down due to the closure of the underlying device.
+	// You don't have to call this.
 	// TODO: return an error?
-	Wait()
+	Wait() chan struct{}
```

Comment:
The conventional type is `<-chan struct{}` like:

https://pkg.go.dev/context#Context.Done

But then might as well call this `Done` to be conventional too. Otherwise it's way too easy to miss & misread a caller writing "foo.Done()" without the receive.

Reply:
Done, thank you!